docs/usage/operationalisation_checklist.md

```diff
@@ -0,0 +1,290 @@
+
+# How do you log what users accessed what resources?
+**Assessment criteria:** onboarding and continued
+**DSS Criteria:** 5
+
+If you application contains resources that are protected, you should record who accessed what, and when, for audit purposes.
+You may be required to archive these audit logs.
+
+# What external systems are you interfacing with?
+**Assessment criteria:** onboarding and continued
+**DSS Criteria:** 4
+
+If your application relies on other systems to deliver its service, what are they and how are they accessed?
+Components like databases and message queues deployed specifically for your application are usually excluded from this list.
+
+## How will you communicate the availability of those external systems to your users?
+If an external system that your application needs to perform its work is performing slowly or is unavailable, how will you inform your users of this?
+
+## Do these external systems require credentials?
+If those external systems require credentials or other security tokens for authentication, how will those credentials be protected and managed by your application?
+
+# Do you use MySQL or PostgreSQL for your database?
+**Assessment criteria:** onboarding and continued
+**DSS Criteria:** 4
+
+cloud.gov.au, supports the following databases:
+
+- MySQL
+- PostgresSQL
+
+If you are planning to use a database that is not on this list you should discuss this with the cloud.gov.au team by sending a support request to support@cloud.gov.au.
+
+## How much data do you plan to store in your database?
+
+# Do you use RabbitMQ for queuing?
+**Assessment criteria:** onboarding and continued
+**DSS Criteria:** 4
+
+cloud.gov.au, supports the following message queues:
+
+- RabbitMQ
+
+If you are planning to use a message queue that is not on this list you should discuss this with the cloud.gov.au team by sending a support request to support@cloud.gov.au.
+
+## How much data do you plan to store in your queue?
+
+# Do you use memcache or Redis for caching?
+**Assessment criteria:** onboarding and continued
+**DSS Criteria:** 4
+
+cloud.gov.au, supports the following caching products:
+
+- memcache
+- Redis
+
+If you are planning to use a cache that is not on this list you should discuss this with the cloud.gov.au team by sending a support request to support@cloud.gov.au.
+
+# How much data do you plan to store in your cache?
+
+# Are users able to upload files? Where will you store them?
+**Assessment criteria:** onboarding and continued
+**DSS Criteria:** 4
+
+If your application allows users to upload files, where do you plan to store this data, and how much of it will there be?
+
+# What is the uptime requirement of your application?
+**Assessment criteria:** onboarding and continued
+**DSS Criteria:** 4
+
+What is the uptime requirement for your application?
+
+## What failure conditions do you wake people up for?
+
+If you application is experiencing a failure, who will be notified and how?
```

Comment:
We need to explain a bit more about the conditions that have to be met to send an alert.